### PR TITLE
Add data sources section

### PR DESCRIPTION
**Pull request by mosabua**
Note for reviewers:

* check the preview deployment or run locally
* the text descriptions for the data sources are all from the original website so we should mostly leave them as they are.

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -1,6 +1,48 @@
...
+  category: data-source
+  logo: /assets/images/logos/amazon-kinesis.png
+  logosmall: /assets/images/logos/amazon-kinesis-small.png
+  description: |
+    tbd
+
+    Use a Amazon Kinesis database as a data source in Trino by configuring a catalog with the Amazon Kinesis connector.
+  links:
+    - urltext: Amazon Kinesis
+      url:
```
Comment (mosabua) [inline _data/tools.yml:15]:
```suggestion
      url: https://aws.amazon.com/kinesis/
```

File: _data/tools.yml [hunk 2]
```diff
@@ -14,8 +67,201 @@
...
+      url: https://trino.io/episodes/40.html
+# there are lots more from Trino Fest, Trino Summot .. need to figure out how many we are happy to have.. 
+- name: Apache Ignite
+  anchor: apache-ignite
+  category: data-source
+  logo: /assets/images/logos/apache-ignite.png
+  logosmall: /assets/images/logos/apache-ignite-small.png
+  description: |
+    Apache Ignite is a distributed database for high‑performance applications
+    with in‑memory speed. It scales across memory and disk across multiople
```
Comment (bitsondatadev) [inline _data/tools.yml:180]:
```suggestion
    with in‑memory speed. It scales across memory and disk across multiple
```

File: _data/tools.yml [hunk 3]
```diff
@@ -14,8 +67,201 @@
...
+      url: https://kafka.apache.org/
+    - urltext: Apache Kafka connector documentation
+      url: https://trino.io/docs/current/connector/kafka.html
+- name: Apache Kudu
+  anchor: apache-kudu
+  category: data-source
+  logo: /assets/images/logos/apache-kudu.png
+  logosmall: /assets/images/logos/apache-kudu-small.png
+  description: |
+    Apache Kudu is an open source distributed data storage engine that makes
```
Comment (bitsondatadev) [inline _data/tools.yml:216]:
```suggestion
    Apache Kudu is an open-source distributed data storage engine that makes
```

Not sure if this needs to be addressed in this PR but I think we should do a mass search for commonly hyphenated words and pick a side. 😄

File: _data/tools.yml [hunk 4]
```diff
@@ -14,8 +67,201 @@
...
+  description: |
+    Apache Phoenix enables OLTP and operational analytics in Hadoop for low
+    latency applications by combining the best of both worlds:
+
+    * The power of standard SQL and JDBC APIs with full ACID transaction
+      capabilities and
+    * The flexibility of late-bound, schema-on-read
+      capabilities from the NoSQL world by leveraging HBase as its backing store
+
+    Use a Apache Phoenix data lake as a data source in Trino by configuring a
```
Comment (bitsondatadev) [inline _data/tools.yml:240]: Phoenix is not a data lake last time I checked 😁


```suggestion
    Use a Apache Phoenix key value store as a data source in Trino by configuring a
```

File: _data/tools.yml [hunk 5]
```diff
@@ -14,8 +67,201 @@
...
+  category: data-source
+  logo: /assets/images/logos/apache-iceberg.png
+  logosmall: /assets/images/logos/apache-iceberg-small.png
+  description: |
+    Apache Iceberg is a high-performance format for huge analytic tables.
+    Iceberg brings the reliability and simplicity of SQL tables to big data,
+    while making it possible for engines like Spark, Trino, Flink, Presto, Hive
+    and Impala to safely work with the same tables, at the same time.
+
+    Use an Apache Iceberg data lakehous as a data source in Trino by configuring
```
Comment (bitsondatadev) [inline _data/tools.yml:163]:
```suggestion
    Use an Apache Iceberg data lakehouse as a data source in Trino by configuring
```

File: _data/tools.yml [hunk 6]
```diff
@@ -109,6 +372,81 @@
...
+  category: data-source
+  logo: /assets/images/logos/elasticsearch.png
+  logosmall: /assets/images/logos/elasticsearch-small.png
+  description: |
+    Elasticsearch is a distributed, RESTful search and analytics engine capable
+    of addressing a growing number of use cases. As the heart of the Elastic
+    Stack, it centrally stores your data for lightning fast search, fine‑tuned
+    relevancy, and powerful analytics that scale with ease.
+
+    Use an Elasticssearch index as a data source in Trino by configuring a
```
Comment (bitsondatadev) [inline _data/tools.yml:410]:
```suggestion
    Use an Elasticssearch search engine as a data source in Trino by configuring a
```

File: _data/tools.yml [hunk 7]
```diff
@@ -54,6 +301,22 @@
...
+- name: Clickhouse
+  anchor: clickhouse
+  category: data-source
+  logo: /assets/images/logos/clickhouse.png
+  logosmall: /assets/images/logos/clickhouse-small.png
+  description: |
+    ClickHouse is the fastest and most resource efficient open-source database
+    for real-time apps and analytics.
+
+    Use a Clickhouse database as a data source in Trino by configuring a catalog
```
Comment (bitsondatadev) [inline _data/tools.yml:313]:
```suggestion
    Use a Clickhouse real-time database as a data source in Trino by configuring a catalog
```

File: _data/tools.yml [hunk 8]
```diff
@@ -14,8 +67,201 @@
...
+- name: Apache Pinot
+  anchor: apache-pinot
+  category: data-source
+  logo: /assets/images/logos/apache-pinot.png
+  logosmall: /assets/images/logos/apache-pinot-small.png
+  description: |
+    Apache Pinot is a realtime distributed OLAP datastore, designed to answer
+    OLAP queries with low latency
+
+    Use an Apache Pinot datastore as a data source in Trino by configuring a
```
Comment (bitsondatadev) [inline _data/tools.yml:256]:
```suggestion
    Use an Apache Pinot real-time database as a data source in Trino by configuring a
```

File: _data/tools.yml [hunk 9]
```diff
@@ -14,8 +67,201 @@
...
+  anchor: apache-kafka
+  category: data-source
+  logo: /assets/images/logos/apache-kafka.png
+  logosmall: /assets/images/logos/apache-kafka-small.png
+  description: |
+    Apache Kafka is an open-source distributed event streaming platform used by
+    thousands of companies for high-performance data pipelines, streaming
+    analytics, data integration, and mission-critical applications.
+
+    Use an Apache Kafka topic stream as a data source in Trino by configuring a
```
Comment (bitsondatadev) [inline _data/tools.yml:203]:
```suggestion
    Use an Apache Kafka event streaming platform as a data source in Trino by configuring a
```

File: _data/tools.yml [hunk 10]
```diff
@@ -14,8 +67,201 @@
...
+  anchor: apache-ignite
+  category: data-source
+  logo: /assets/images/logos/apache-ignite.png
+  logosmall: /assets/images/logos/apache-ignite-small.png
+  description: |
+    Apache Ignite is a distributed database for high‑performance applications
+    with in‑memory speed. It scales across memory and disk across multiople
+    machines without compromise.
+
+    Use an Apache Ignite database as a data source in Trino by configuring a
```
Comment (bitsondatadev) [inline _data/tools.yml:183]:
```suggestion
    Use an Apache Ignite in-memory database as a data source in Trino by configuring a
```

File: _data/tools.yml [hunk 11]
```diff
@@ -14,8 +67,201 @@
...
+- name: Apache Druid
+  anchor: apache-druid
+  category: data-source
+  logo: /assets/images/logos/apache-druid.png
+  logosmall: /assets/images/logos/apache-druid-small.png
+  description: |
+    Druid is a high performance, real-time analytics database that delivers
+    sub-second queries on streaming and batch data at scale and under load.
+
+    Use an Apache Druid database as a data source in Trino by configuring a
```
Comment (bitsondatadev) [inline _data/tools.yml:101]:
```suggestion
    Use an Apache Druid real-time database as a data source in Trino by configuring a
```

File: _data/tools.yml [hunk 12]
```diff
@@ -14,8 +67,201 @@
...
+  logo: /assets/images/logos/apache-cassandra.png
+  logosmall: /assets/images/logos/apache-cassandra-small.png
+  description: |
+    Apache Cassandra is an open source NoSQL distributed database trusted by
+    thousands of companies for scalability and high availability without
+    compromising performance. Linear scalability and proven fault-tolerance on
+    commodity hardware or cloud infrastructure make it the perfect platform for
+    mission-critical data.
+
+    Use an Apache Cassandra database as a data source in Trino by configuring a
```
Comment (bitsondatadev) [inline _data/tools.yml:85]:
```suggestion
    Use an Apache Cassandra columnar database as a data source in Trino by configuring a
```

File: _data/tools.yml [hunk 13]
```diff
@@ -1,6 +1,59 @@
...
+- name: Apache Accumulo
+  anchor: apache-accumulo
+  category: data-source
+  logo: /assets/images/logos/apache-accumulo.png
+  logosmall: /assets/images/logos/apache-accumulo-small.png
+  description: |
+    Apache Accumulo® is a sorted, distributed key/value store that provides
+    robust, scalable data storage and retrieval.
+
+    Use an Apache Accumulo cluster as a data source in Trino by configuring a
```
Comment (bitsondatadev) [inline _data/tools.yml:50]:
```suggestion
    Use an Apache Accumulo key-value store as a data source in Trino by configuring a
```

File: _data/tools.yml [hunk 14]
```diff
@@ -231,6 +570,59 @@
...
+  category: data-source
+  logo: /assets/images/logos/mariadb.png
+  logosmall: /assets/images/logos/mariadb-small.png
+  description: |
+    MariaDB Server is one of the most popular open source relational databases.
+    It’s made by the original developers of MySQL and guaranteed to stay open
+    source. It is part of most cloud offerings and the default in most Linux
+    distributions.
+
+    Use a MariaDB database as a data source in Trino by configuring a catalog
```
Comment (bitsondatadev) [inline _data/tools.yml:586]:
```suggestion
    Use a MariaDB relational database as a data source in Trino by configuring a catalog
```

File: _data/tools.yml [hunk 15]
```diff
@@ -231,6 +570,59 @@
...
+  anchor: mongodb
+  category: data-source
+  logo: /assets/images/logos/mongodb.png
+  logosmall: /assets/images/logos/mongodb-small.png
+  description: |
+    MongoDB is a source-available cross-platform document-oriented database
+    program. Classified as a NoSQL database program, MongoDB uses JSON-like
+    documents with optional schemas.
+
+    Use a MongoDB database as a data source in Trino by configuring a catalog
```
Comment (bitsondatadev) [inline _data/tools.yml:603]:
```suggestion
    Use a MongoDB document database as a data source in Trino by configuring a catalog
```

File: _data/tools.yml [hunk 16]
```diff
@@ -231,6 +570,59 @@
...
+- name: MySQL
+  anchor: mysql
+  category: data-source
+  logo: /assets/images/logos/mysql.png
+  logosmall: /assets/images/logos/mysql-small.png
+  description: |
+    MySQL is the world's most popular open-source relational database management
+    system (RDBMS).
+
+    Use a MySQL database as a data source in Trino by configuring a catalog with
```
Comment (bitsondatadev) [inline _data/tools.yml:619]:
```suggestion
    Use a MySQL relational database as a data source in Trino by configuring a catalog with
```

File: _data/tools.yml [hunk 17]
```diff
@@ -261,6 +653,23 @@
...
+  anchor: oracle
+  category: data-source
+  logo: /assets/images/logos/oracle.png
+  logosmall: /assets/images/logos/oracle-small.png
+  description: |
+    Oracle database services and products offer customers cost-optimized and
+    high-performance versions of Oracle Database, the world's leading converged,
+    multi-model database management system.
+
+    Use an Oracle database as a data source in Trino by configuring a catalog
```
Comment (bitsondatadev) [inline _data/tools.yml:666]:
```suggestion
    Use an Oracle relational database as a data source in Trino by configuring a catalog
```

File: _data/tools.yml [hunk 18]
```diff
@@ -274,6 +683,24 @@
...
+  category: data-source
+  logo: /assets/images/logos/postgresql.png
+  logosmall: /assets/images/logos/postgresql-small.png
+  description: |
+    PostgreSQL is the world's most advanced open source relational database.
+    PostgreSQL is a powerful system with over 35 years of active development
+    that has earned it a strong reputation for reliability, feature robustness,
+    and performance.
+
+    Use a PostgreSQL database as a data source in Trino by configuring a catalog
```
Comment (bitsondatadev) [inline _data/tools.yml:697]:
```suggestion
    Use a PostgreSQL relational database as a data source in Trino by configuring a catalog
```

File: _data/tools.yml [hunk 19]
```diff
@@ -286,6 +713,25 @@
   links:
     - urltext: Presto-Gateway
       url: https://github.com/lyft/presto-gateway
+- name: Prometheus
+  anchor: prometheus
+  category: data-source
+  logo: /assets/images/logos/prometheus.png
+  logosmall: /assets/images/logos/prometheus-small.png
+  description: |
+    Prometheus is an open-source systems monitoring and alerting toolkit with  a
```
Comment (bitsondatadev) [inline _data/tools.yml:722]:
```suggestion
    Prometheus is an open-source systems monitoring and alerting toolkit with a
```

File: _data/tools.yml [hunk 20]
```diff
@@ -342,6 +788,22 @@
   links:
     - urltext: Redash
       url: https://redash.io/
+- name: Redis
+  anchor: redis
+  category: data-source
+  logo: /assets/images/logos/redis.png
+  logosmall: /assets/images/logos/redis-small.png
+  description: |
+    Redis is an  open source, in-memory data store used by millions of
```
Comment (bitsondatadev) [inline _data/tools.yml:797]:
```suggestion
    Redis is an open-source, in-memory data store used by millions of
```

File: _data/tools.yml [hunk 21]
```diff
@@ -342,6 +788,22 @@
...
+    developers as a database, cache, streaming engine, and message broker.
+
+    Use a Redis database as a data source in Trino by configuring a catalog with
```
Comment (bitsondatadev) [inline _data/tools.yml:800]:
```suggestion
    Use a Redis in-memory database as a data source in Trino by configuring a catalog with
```

File: _data/tools.yml [hunk 22]
```diff
@@ -1,6 +1,59 @@
...
+      url: https://aws.amazon.com/redshift/
+    - urltext: Amazon Redshift connector documentation
+      url: https://trino.io/docs/current/connector/redshift.html
+- name: Apache Accumulo
+  anchor: apache-accumulo
+  category: data-source
+  logo: /assets/images/logos/apache-accumulo.png
+  logosmall: /assets/images/logos/apache-accumulo-small.png
+  description: |
+    Apache Accumulo® is a sorted, distributed key/value store that provides
```
Comment (bitsondatadev) [inline _data/tools.yml:47]:
```suggestion
    Apache Accumulo® is a sorted, distributed key-value store that provides
```

**Review by bitsondatadev — CHANGES_REQUESTED** [review]
Looks close!

File: _data/tools.yml [hunk 23]
```diff
@@ -14,8 +67,201 @@
...
+    Use an Apache Iceberg data lakehous as a data source in Trino by configuring
+    a catalog with the Apache Iceberg connector.
+  links:
+    - urltext: Apache Iceberg
+      url: https://iceberg.apache.org/
+    - urltext: Apache Iceberg connector documentation
+      url: https://trino.io/docs/current/connector/iceberg.html
+    - urltext: Interview with Iceberg creator from Trino Community Broadcast 40
+      url: https://trino.io/episodes/40.html
+# there are lots more from Trino Fest, Trino Summot .. need to figure out how many we are happy to have..
```
[the remark this answers is not in the document]
Comment (mosabua) [reply _data/tools.yml:172]: There is too many .. I think we should do a follow up PR around this.

[the remark this answers is not in the document]
Comment (bitsondatadev) [reply _data/tools.yml:172]: Yeah I know haha

File: ecosystem/clients-intro.md [hunk 24]
```diff
@@ -1,7 +1,9 @@
 Clients allow you to connect to Trino, submit SQL queries, and receive the
-results. Clients are full-featured applications or libraries and drivers that
-allow you to connect to any applications supporting that driver or even your own
-custom application or script.
+results. Clients can access all [configured catalogs, and therefore data
```
[the remark this answers is not in the document]
Comment (mosabua) [reply ecosystem/clients-intro.md:2]: Reworded differently to adjust

File: _data/tools.yml (hunk 7 above)
Comment (mosabua) [reply _data/tools.yml:313]: I worked this better into the above paragraph .. this connection sentence should be similar across all of them

File: _data/tools.yml (hunk 13 above)
Comment (mosabua) [reply _data/tools.yml:50]: applied

**Comment by bitsondatadev** [comment]
> I addressed all comments @bitsondatadev  I chose not use the the qualifiers like "relational" in the paragraph about the connectors in any of them since that just repeats what the first paragraph explains .. and it would be kinda random and incomplete, or have to be a full duplicate.. 

Hmm perhaps it would be better to have a category rage at some point that let's you filter based on the category of data source.

**Comment by mosabua** [comment]
I added the missing SQL Server now and used the Trino connector names in the second paragraph and link. I think this should be good to go now @bitsondatadev

**Review by bitsondatadev — APPROVED** [review]
Looks good. Nice catch with SQL Server.

**Pull request merged**